gen/builders/generator.go

```diff
@@ -86,35 +108,56 @@ type MessageVectorGenItem struct {
 
 func NewGenerator() *Generator {
 	// Consume CLI parameters.
-	var (
-		outputDir = flag.String("o", "", "directory where test vector JSON files will be saved; if omitted, vectors will be written to stdout")
-		filter    = flag.String("f", "", "regex filter to select a subset of vectors to execute; matched against the vector's ID")
-	)
+	var outputDir string
+	const outputDirUsage = "directory where test vector JSON files will be saved; if omitted, vectors will be written to stdout."
+	flag.StringVar(&outputDir, "o", "", outputDirUsage)
+	flag.StringVar(&outputDir, "out", "", outputDirUsage)
+
+	var update bool
+	const updateUsage = "update any existing test vector files in the output directory IF their content has changed. Note `_meta` is ignored when checking equality."
+	flag.BoolVar(&update, "u", false, updateUsage)
+	flag.BoolVar(&update, "update", false, updateUsage)
+
+	var force bool
+	const forceUsage = "force regeneration and overwrite any existing vectors in the output directory."
+	flag.BoolVar(&force, "f", false, forceUsage)
+	flag.BoolVar(&force, "force", false, forceUsage)
+
+	var filter string
```

Comment:
Can we rename this variable and review the texts to make sure we talk about an _inclusion_ filter? Wanna make sure things are coherent and relatively frictionless.